lambdas/fetch-questions/tsconfig.json

```diff
@@ -15,5 +15,5 @@
     "noImplicitAny": true
   },
   "exclude": ["node_modules", "build", "coverage"],
-  "include": ["src", "tests"]
+  "include": ["src", "tests", "../../lib/src/Service/metrics-probe.ts"]
```

Comment:
Why do we exclude the metrics probe here?

Reply:
The idea is that metrics-probe.ts will be shared between this and the answers lambda (or others)
This is line just makes sure it is included when the code is compiled.